README.md

```diff
@@ -181,3 +182,40 @@ any outputs.
 | `name`    | Unique name for this rule (required)                                                |
 | `srcs`    | Sass files included in this library. Each file should start with an underscore      |
 | `deps`    | Dependencies for the `src`. Each dependency is a `sass_library`                     |
+
+### multi_sass_binary
+
+```py
+multi_sass_binary(name, srcs=[], output_style="compressed", sourcemap=True)
+```
+
+`multi_sass_binary` compiles a list of Sass files and outputs the corresponding
+CSS files and optional sourcemaps. Output is omitted for filenames that start
+with underscore "_".
+
+
+WARNING: This rule does a global compilation, and thus any change in the sources
```

Comment:
Do we want to note that it's also not used internally at Google? Just so Googlers don't expect to use this rule in third_party code that's synced

Reply:
done.